Frontend/src/Patient/NewPatientModal.jsx

```diff
@@ -1,10 +1,12 @@
 import React, { useState, useEffect } from 'react';
 import { Modal, Row, Col, Table, Form, FormGroup, FormLabel } from 'react-bootstrap';
-import { useUser } from '../UserContext';
+import { UserContext, useUser, setPatientsTabs} from '../UserContext';
 import SearchBarPatient from './SearchBarPatient';
 
 const NewPatientModal = ({ onClose, onCreate}) => {
-  const { user, setUser } = useUser();
+  const { user } = useUser();
+  const [patients, setPatients] = useState([]);
+  const [patientsTabs] = useState([]);
```

Comment:
What is the reason for having [patientsTabs] = useState([]) with no setter? The value of this variable will never change.